### PR TITLE
Add releases docs

### PR DESCRIPTION
**Pull request by tomaszdurka**

### CODE REVIEW

File: docs/Releases.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Above command will:
+1. Add to PRERELEASE-CHANGELOG.md
+2. Push your current branch state to remote branch, create and publish alpha git tag
+3. Every published tag will trigger npm publish
+
+
+### Finalizing release
+
+Once work via pre-releases is done and you want to release actual non-alpha release you need to merge release pull-request into the master branch.
```
Comment (Itrulia) [inline docs/Releases.md:41]: I would add a few helping words to ease the reading flow a la

```suggestion
Once your work via pre-releases is done and you want to release the actual non-alpha release you need to merge the release pull-request into the master branch.
```

File: docs/Releases.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Releases
+
+## Versioning
+
+We use semantic versioning to indicate changes within our public API.
```
Comment (Itrulia) [inline docs/Releases.md:5]: Adding a link to semver.org could be helpful

```suggestion
We use [semantic versioning]((https://semver.org/)) to indicate changes within our public API.
```

File: docs/Releases.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Above command will:
+1. Add to PRERELEASE-CHANGELOG.md
+2. Push your current branch state to remote branch, create and publish alpha git tag
+3. Every published tag will trigger npm publish
+
+
+### Finalizing release
+
+Once work via pre-releases is done and you want to release actual non-alpha release you need to merge release pull-request into the master branch.
+Above merge will trigger action that will:
```
Comment (Itrulia) [inline docs/Releases.md:42]:
```suggestion
Above merge will trigger actions that will:
```

**Comment by Itrulia** [comment]
Looking good! 👍🏼 
Just added a few suggestions.

**Comment by tomaszdurka** [comment]
@elias-summermatter waiting on your approval based on your experience with `yarn release prerelease` for `3.0.0`.

**Comment by tomaszdurka** [comment]
Closing in favor of #592

**Pull request closed without merging**